templates/notebook_instance_lifecycle_config/sdk_update_post_set_output.go.tpl

```diff
@@ -0,0 +1,6 @@
+//Done because controller finishes reconciling after update.
+updated, err = rm.sdkFind(ctx, latest)
```

Comment:
instead of updating one value, please requeue so everything can be synced and validated from the service side

tomorrow if there is a bug in delta, we will be able to catch it